common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -171,18 +169,14 @@ public void run() {
     @Override
     public boolean onBackPressed() {
         Logger.info(TAG, "Back button is pressed");
-        if (null != mWebView && mWebView.canGoBack()) {
-            // User should be able to click back button to cancel. Counting blank page as well.
-            final int BACK_PRESSED_STEPS = -2;
-            if (!mWebView.canGoBackOrForward(BACK_PRESSED_STEPS)) {
-                cancelAuthorization(true);
-            } else {
-                mWebView.goBack();
-            }
-            return true;
+
+        if (mWebView.canGoBack()) {
+            mWebView.goBack();
+        } else {
+            cancelAuthorization(true);
         }
 
-        return false;
+        return true;
```

Comment:
This used to return true iff we had a WebView and it could go back, false otherwise.  Do we need to?

Reply:
Nope, unclear to me why this was ever written this way -- `false` should be used by this Fragment to signal "I did not handle this event" but, if it's on-screen, there is _always_ something for this Fragment to do: either `goBack()` or self-cancel. Irrespective of which happens we should `return true`, signaling we handled the `onBackPressed()` event ourselves